django_celery_results/urls.py

```diff
@@ -6,21 +6,49 @@
     URL  to :func:`~celery.views.task_status`.
 """
 
-from django.conf.urls import url
+from django.conf.urls import path, register_converter
 
 from . import views
 
-task_pattern = r'(?P<task_id>[\w\d\-\.]+)'
+
+class TaskPatternConverter:
```

Comment:
so  we still need a custom converter?

Reply:
I believe so. Django has a `uuid` converter, but the pattern we're using is a little different. Specifically, our pattern allows for `.`, while uuid does not. For what its worth I've never seen a celery task ID with a period, but I haven't found the official docs on that

Reply:
@AlecTroemel the task id is provided by `kombu.uuid()` which is an alias for the [builtin uuid4](https://docs.python.org/dev/library/uuid.html#uuid.uuid4).
[according to wikipedia](https://en.wikipedia.org/wiki/Universally_unique_identifier#Format) it will only contain HEX characters and hyphens (`[0-1a-zA-Z\-]`) So I think the custom converter is not needed if it's only used for the UUID pattern

Reply:
@AllexVeldman I don't disagree with you, however the old task pattern does allow for periods `.`.  Again I've never actually seen a period in a task ID, so it could be that there never have been and  never will be any. If that's the case I'd love to use the default uuid converter! 

I'll do some digging in the commit history to see if I can learn anything about the context of the old pattern

Reply:
hmm, not a whole lot of history with this file https://github.com/celery/django-celery-results/commits/master/django_celery_results/urls.py